pkg/client/client_test.go

```diff
@@ -269,8 +353,24 @@ var _ = Describe("Client", func() {
 
 		})
 
-		It("should fail if the object cannot be mapped to a GVK", func() {
+		It("should fail if the object cannot be mapped to a GVK", func(done Done) {
+			By("creating client with empty Scheme")
+			emptyScheme := runtime.NewScheme()
+			cl, err := client.New(cfg, client.Options{Scheme: emptyScheme})
```

Comment:
same comment as earlier about not specifying the scheme.

Reply:
I think it makes it easier to check for things not in the Scheme if an empty one is passed in

Reply:
For this one, I'll keep the empty scheme.